curriculum/challenges/english/25-front-end-development/lab-lightbox-viewer/66db57ad34c7089b9b41bfd6.md

```diff
@@ -234,6 +234,30 @@ background.dispatchEvent(new Event('click'));
 assert.strictEqual(getComputedDisplay(lightbox), 'none');
 ```
 
+When your `.lightbox` element is visible and you click the `.lightbox` element, the `.lightbox` elements `display` should be set back to `none`.
+
+```js
+// Get the lightbox element for simulating clicks outside
+const lightbox = document.querySelector(".lightbox");
+
+// Function to get the computed display property
+function getComputedDisplay(element) {
+  return window.getComputedStyle(element).display;
+}
+
+// Make sure the lightbox is visible initially
+assert.strictEqual(getComputedDisplay(lightbox), 'flex');
```

Comment:
here, and in the other test, instead of asserting, let's set the `display` property to `flex` so we are sure it's open. If it's closed to start the tests will fail